src/Evolution/Systems/NewtonianEuler/Subcell/NeighborPackagedData.hpp

```diff
@@ -0,0 +1,216 @@
+ * subcells, with the correction being projected back to the DG interface.
+ * However, in practice such strict conservation doesn't seem to be necessary
+ * and can be explained by that we only need strict conservation at shocks, and
+ * if one element is doing DG, then we aren't at a shock.
+ */
+struct NeighborPackagedData {
+  template <size_t Dim, typename DbTagsList>
+  static FixedHashMap<maximum_number_of_neighbors(Dim),
+                      std::pair<Direction<Dim>, ElementId<Dim>>,
+                      std::vector<double>,
+                      boost::hash<std::pair<Direction<Dim>, ElementId<Dim>>>>
+  apply(const db::DataBox<DbTagsList>& box,
+        const std::vector<std::pair<Direction<Dim>, ElementId<Dim>>>&
+            mortars_to_reconstruct_to) {
+    using system =
+        typename std::decay_t<decltype(db::get<Parallel::Tags::Metavariables>(
+            box))>::system;
+    using evolved_vars_tag = typename system::variables_tag;
+    using evolved_vars_tags = typename evolved_vars_tag::tags_list;
+    using prim_tags = typename system::primitive_variables_tag::tags_list;
+    using fluxes_tags = db::wrap_tags_in<::Tags::Flux, evolved_vars_tags,
+                                         tmpl::size_t<Dim>, Frame::Inertial>;
+
+    ASSERT(not db::get<domain::Tags::MeshVelocity<Dim>>(box).has_value(),
+           "Haven't yet added support for moving mesh to DG-subcell. This "
+           "should be easy to generalize, but we will want to consider "
+           "storing the mesh velocity on the faces instead of "
+           "re-slicing/projecting.");
+
+    FixedHashMap<maximum_number_of_neighbors(Dim),
+                 std::pair<Direction<Dim>, ElementId<Dim>>, std::vector<double>,
+                 boost::hash<std::pair<Direction<Dim>, ElementId<Dim>>>>
+        nhbr_package_data{};
+
+    const auto& nhbr_subcell_data =
+        db::get<evolution::dg::subcell::Tags::
+                    NeighborDataForReconstructionAndRdmpTci<Dim>>(box);
+    const Mesh<Dim>& subcell_mesh =
+        db::get<evolution::dg::subcell::Tags::Mesh<Dim>>(box);
+    const Mesh<Dim>& dg_mesh = db::get<domain::Tags::Mesh<Dim>>(box);
+
+    const auto volume_prims = evolution::dg::subcell::fd::project(
+        db::get<typename system::primitive_variables_tag>(box), dg_mesh,
+        subcell_mesh.extents());
+
+    const auto& recons =
+        db::get<NewtonianEuler::fd::Tags::Reconstructor<Dim>>(box);
+    const auto& boundary_correction =
+        db::get<evolution::Tags::BoundaryCorrection<system>>(box);
+    using derived_boundary_corrections =
+        typename std::decay_t<decltype(boundary_correction)>::creatable_classes;
+    tmpl::for_each<
+        derived_boundary_corrections>([&box, &boundary_correction, &dg_mesh,
+                                       &mortars_to_reconstruct_to,
+                                       &nhbr_package_data, &nhbr_subcell_data,
```

Comment:
I think we've been trying to spell out `neighbor` instead of using `nhbr` (my fault, sorry)